src/sbd_helper.in

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+
+# Authors:
+#  Klaus Wenninger <kwenning@redhat.com>
+#
+# License: Revised BSD
+
+# chkconfig: - 99 01
+# description: Shared-storage based fencing daemon
+# processname: sbd
+#
+### BEGIN INIT INFO
+# Provides:		sbd_helper
+# Required-Start:	$local_fs $remote_fs
+# Should-Start:		$syslog
+# Required-Stop:	$local_fs $remote_fs
```

Comment:
I think we also need:

Should-Stop: $syslog
X-Start-Before: corosync pacemaker_remote
X-Stop-After: corosync pacemaker_remote

Possibly add iscsi to Should-Start/Stop too, if the package name is reasonably consistent among distros